test.sh

```diff
@@ -20,8 +20,9 @@ LUA=../src/lua
 
 for serialise in 0 1; do
     for test in api bwcoercion closure code events \
-                gengc pm tpack tracegc vararg goto cstack locals; do
+                gengc pm tpack tracegc vararg goto \
+                cstack locals files; do
         echo "### YKD_SERIALISE_COMPILATION=$serialise $test.lua ###"
-        YKD_SERIALISE_COMPILATION=$serialise ${LUA} ${test}.lua
+        YKD_SERIALISE_COMPILATION=$serialise ${LUA} -e"_U=true" ${test}.lua
```

Comment:
I'm surprised we didn't have `_U=true` before! Isn't that necessary? Should it be a separate commit?

Reply:
yeah I will split into separate commits 
We had `_U=true` in this stake PR: https://github.com/ykjit/yklua/pull/69

I think "_U" is only used in `all.lua` test suite  (https://github.com/ykjit/yklua/blob/main/tests/all.lua#L28) and we're not using it currently.